telemetry/definitions/commonDefinitions.json

```diff
@@ -212,6 +212,11 @@
             "allowedValues": ["ec2", "fargate"],
             "description": "Infrastructure type used by ECS tasks and services"
         },
+        {
+            "name": "ecsExecuteCommandOrigin",
```

Comment:
nit: I think "commandType" is a better name for this.

Reply:
It's less about the "origin" and more about the type of activity the customer is taking.

Reply:
Agreed, name changed